testing/perlmagick/APKBUILD

```diff
@@ -0,0 +1,82 @@
+# Contributor: Timothy Legge <timlegge@gmail.com>
+# Maintainer: Timothy Legge <timlegge@gmail.com>
+# Natanael Copa <ncopa@alpinelinux.org>
+pkgname=perlmagick
+pkgver=7.0.8.39
+pkgrel=0
+_pkgver=${pkgver%.*}-${pkgver##*.}
+_abiver=7
+pkgdesc="Perl Module for Image Magick"
+url="https://www.imagemagick.org/"
+arch="all"
+license="ImageMagick"
+options="libtool !checkroot"
+depends="imagemagick imagemagick-libs imagemagick-c++"
+makedepends="perl-dev libtool chrpath"
+checkdepends=""
+subpackages="$pkgname-doc"
+source="https://www.imagemagick.org/download/releases/ImageMagick-$_pkgver.tar.xz"
+builddir="$srcdir/ImageMagick-${_pkgver}"
+
+build() {
+	cd "$builddir"
+	# fix doc dir, Gentoo bug 91911
+	sed -i -e \
+		's:DOCUMENTATION_PATH="${DATA_DIR}/doc/${DOCUMENTATION_RELATIVE_PATH}":DOCUMENTATION_PATH="/usr/share/doc/imagemagick":g' \
+		configure
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr \
+		--sysconfdir=/etc \
+		--mandir=/usr/share/man \
+		--infodir=/usr/share/info \
+		--disable-static \
+		--disable-openmp \
+		--with-threads \
+		--with-x \
+		--with-tiff \
+		--with-png \
+		--with-webp \
+		--with-rsvg \
+		--with-gslib \
+		--with-gs-font-dir=/usr/share/fonts/Type1 \
+		--with-modules \
+		--with-xml \
+		--with-perl \
+		--with-perl-options=PREFIX=/usr \
+		$_pic
+	make perl-sources
+	make perl-build
+	cd "$builddir/PerlMagick"
+	chrpath -d ./blib/arch/auto/Image/Magick/Q16HDRI/Q16HDRI.so
+}
+
+check() {
+	cd "$builddir/PerlMagick"
+	if ! [ -e blib/lib/Image/Magick.pm ]; then
+		error "Module does not exist Magick.pm"
+		return 1
+	fi
+        if ! [ -e blib/lib/Image/Magick/Q16HDRI.pm ]; then
```

Comment:
Inconsistent whitespace. Please use tabs for indentation.

Reply:
oops.  Will do